### PR TITLE
Deduplicated installation documentation

### PR DESCRIPTION
**Pull request by JohannesGaessler**
Fixes https://github.com/dsavoiu/kafe2/issues/136 .

@dsavoiu , if you decide to merge this PR, please update the readthedocs page.

### CODE REVIEW

**Comment by Citiroller** [comment]
Looks good to me. Inside the readme we can link to the installation guide on our documentation on readthedocs.

File: README.rst [hunk 1]
```diff
@@ -24,184 +20,15 @@ data. It is the spiritual successor to the original *kafe* package.
...
-
-    apt-get install libpython2-dev libpython3-dev
-
-In Fedora/RHEL/CentOS, do:
-
-.. code:: bash
-
-    yum install python2-devel python3-devel
-
+A user guide, including installation instructions, can be found under `kafe2.readthedocs.io <https://kafe2.readthedocs.io/en/latest/parts/user_guide.html>`_.
```
Comment (Citiroller) [inline README.rst:34]: The installation instructions are located at [https://kafe2.readthedocs.io/en/latest/parts/installation.html](https://kafe2.readthedocs.io/en/latest/parts/installation.html).

I plan to refactor the current "User Guide" to "Beginners Guide" and create a "User Guide" with more specific information on how to do certain things. E.g. enable logging, customize the plots, using the ContoursProfiler etc.

**Comment by Citiroller** [comment]
Would it be ok, if I merge this PR into the doc branch and then merge all doc updates from this one branch?

**Comment by JohannesGaessler** [comment]
I see no problem with doing that; I only made pull requests because these aren't urgent changes and I wanted to get feedback.

**Comment by Citiroller** [comment]
Lives on in #145

**Pull request closed without merging**